src/main/resources/messages/en.yml

```diff
@@ -186,7 +186,7 @@ ChannelTopicUpdaterConsoleChannelTopicAtServerShutdownFormat: "Server is offline
 # Discord command message
 # This is the message sent to players when they run "/discord". It's recommended to leave command syntax as a part of this
 #
-DiscordCommandFormat: "&bJoin us on Discord at http://discord.gg/changethisinmessages.yml. For command help, do \"/discord ?\""
+DiscordCommandFormat: "&bJoin us on Discord at {INVITE}. For command help, do \"/discord ?\""
```

Comment:
Add explanation of {INVITE} in the comment & that it's a config option in the same file (in some logical, user-friendly way)